buildenv/Build_Instructions_V8.md

```diff
@@ -411,12 +437,24 @@ Run:
 
 Here is some sample output:
```

Comment:
The `cd build/windows-x86_64-normal-server-release/images/j2sdk-image` line above is only appropriate for 64-bit

Reply:
Already updated this section